Core/StorageCache.swift

```diff
@@ -64,17 +70,23 @@ public class StorageCache: StorageCacheUpdating {
             return fileStore.persist(data as? Data, forConfiguration: configuration)
             
         case .trackerDataSet:
-            if fileStore.persist(data as? Data, forConfiguration: configuration) {
-                if TrackerDataManager.shared.reload(etag: etag) != .downloaded {
-                    Pixel.fire(pixel: .trackerDataReloadFailed)
-                    return false
-                }
-                return true
+            if fileStore.persist(data as? Data, forConfiguration: configuration),
+               TrackerDataManager.shared.reload(etag: etag) != .downloaded {
+                Pixel.fire(pixel: .trackerDataReloadFailed)
+                return false
             }
-            return false
+            return true
+            
+        case .privacyConfiguration:
```

Comment:
@bwaresiak there is now a cyclomatic complexity warning in this method. I did some things to reduce this (see the if statements) but the lowest I can get it is 11. Not sure if you have any ideas here.